app/src/main/java/com/codingblocks/cbonlineapp/mycourse/codechallenge/CodeChallengeActivity.kt

```diff
@@ -0,0 +1,88 @@
+package com.codingblocks.cbonlineapp.mycourse.codechallenge
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.TextView
+import androidx.core.view.isVisible
+import com.codingblocks.cbonlineapp.R
+import com.codingblocks.cbonlineapp.analytics.AppCrashlyticsWrapper
+import com.codingblocks.cbonlineapp.util.*
+import com.codingblocks.cbonlineapp.util.extensions.observer
+import com.codingblocks.cbonlineapp.util.extensions.setToolbar
+import com.codingblocks.onlineapi.ErrorStatus
+import com.codingblocks.cbonlineapp.util.extensions.showSnackbar
+import com.google.android.material.snackbar.Snackbar
+import kotlinx.android.synthetic.main.activity_code_challenge.*
+import org.koin.androidx.viewmodel.ext.android.stateViewModel
+
+class CodeChallengeActivity : AppCompatActivity() {
+
+    private val vm: CodeChallengeViewModel by stateViewModel()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_code_challenge)
+        setToolbar(toolbarCodeChallenge)
+        title=""
```

Comment:
why the empty title?

Reply:
Otherwise it shows app name while data is loading. So to show empty it is set empty in starting